docusaurus.config.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
       logo: {
         alt: "Karmada",
         src: "img/logo.svg",
-        srcDark: "img/logoDark.svg",
+        srcDark: "img/logoWhite.svg",
```

Comment:
```suggestion
        srcDark: "img/artwork/karmada-icon-white.svg",
```
I'd prefer to put this image in a separate directory and rename it.